QBot/telegram_bot/views/error.py

```diff
@@ -0,0 +1,25 @@
+from telegram import Update, ReplyKeyboardMarkup
+from telegram.ext import (
+    CallbackContext,
+    ConversationHandler,
+    Filters,
+    MessageHandler,
+)
+from .start import reply_keyboard
+
+
+def wrong_message(update: Update, context: CallbackContext):
+    chat_id = update.message.chat.id
+    context.bot.send_message(chat_id, ("invalid option"))
+    return
+
+
+def cancel(update: Update, context: CallbackContext):
+    chat_id = update.message.chat.id
+    context.bot.send_message(
+        chat_id,
+        text="cancel",
+        reply_markup=ReplyKeyboardMarkup(reply_keyboard, one_time_keyboard=True),
```

Comment:
Using other keyboards in the cancel handler is not well.
besides, we should log the errors!

but the main idea is good 👍

Reply:
If you need to find more search for `services` in the Django and `logger`

Reply:
We don't need errors anymore we can remove them 👍 
we handle errors in `services.py` and we add it to the bot in `telegrambot.py` to find out more check that out =)